<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>975</width>
    <height>589</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>宋体</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>YQ的仿抖音项目</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/images/images/Titel.png</normaloff>:/images/images/Titel.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(22, 24, 35);
</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="spacing">
     <number>1</number>
    </property>
    <property name="leftMargin">
     <number>1</number>
    </property>
    <property name="topMargin">
     <number>1</number>
    </property>
    <property name="rightMargin">
     <number>1</number>
    </property>
    <property name="bottomMargin">
     <number>1</number>
    </property>
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string/>
      </property>
      <widget class="TMyVideoWidget" name="videoWidget" native="true">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>60</y>
         <width>761</width>
         <height>371</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
      </widget>
      <widget class="QWidget" name="widget" native="true">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>440</y>
         <width>761</width>
         <height>35</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>35</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>25</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(175, 178, 178);
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(175, 178, 178, 0), stop:1 rgba(175, 178, 178, 120));
border-color:  rgb(22, 24, 35);</string>
       </property>
       <widget class="QSlider" name="sliderPosition">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>10</y>
          <width>581</width>
          <height>18</height>
         </rect>
        </property>
        <property name="tracking">
         <bool>false</bool>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QLabel" name="labCurMedia">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>10</y>
          <width>100</width>
          <height>19</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Microsoft YaHei UI</family>
          <pointsize>8</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(248, 248, 248);</string>
        </property>
        <property name="text">
         <string>无曲目</string>
        </property>
       </widget>
      </widget>
      <widget class="QPushButton" name="BtnUP">
       <property name="geometry">
        <rect>
         <x>910</x>
         <y>160</y>
         <width>41</width>
         <height>61</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="res.qrc">
         <normaloff>:/images/images/UP.png</normaloff>:/images/images/UP.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
      </widget>
      <widget class="QPushButton" name="BtnDown">
       <property name="geometry">
        <rect>
         <x>910</x>
         <y>220</y>
         <width>41</width>
         <height>61</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="res.qrc">
         <normaloff>:/images/images/DOWN.png</normaloff>:/images/images/DOWN.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
      </widget>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>480</y>
         <width>761</width>
         <height>61</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(175, 178, 178);
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(175, 178, 178, 0), stop:1 rgba(175, 178, 178, 120));
border-color:  rgb(22, 24, 35);</string>
       </property>
       <property name="title">
        <string/>
       </property>
       <widget class="QPushButton" name="btnPlay">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>0</y>
          <width>51</width>
          <height>71</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="res.qrc">
          <normaloff>:/images/images/BF.png</normaloff>:/images/images/BF.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QLabel" name="LabRatio">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>20</y>
          <width>91</width>
          <height>31</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>0</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Microsoft YaHei UI</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(248, 248, 248);
</string>
        </property>
        <property name="text">
         <string>进度</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btnSound">
        <property name="geometry">
         <rect>
          <x>320</x>
          <y>0</y>
          <width>53</width>
          <height>61</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="res.qrc">
          <normaloff>:/images/images/YP.png</normaloff>:/images/images/YP.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QSlider" name="sliderVolumn">
        <property name="geometry">
         <rect>
          <x>390</x>
          <y>20</y>
          <width>160</width>
          <height>20</height>
         </rect>
        </property>
        <property name="maximumSize">
         <size>
          <width>160</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="value">
         <number>100</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QComboBox" name="comboBoxBS">
        <property name="geometry">
         <rect>
          <x>570</x>
          <y>20</y>
          <width>89</width>
          <height>27</height>
         </rect>
        </property>
        <item>
         <property name="text">
          <string>倍速</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1.00</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1.25</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1.50</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2.00</string>
         </property>
        </item>
       </widget>
       <widget class="QPushButton" name="pushButton">
        <property name="geometry">
         <rect>
          <x>690</x>
          <y>10</y>
          <width>61</width>
          <height>61</height>
         </rect>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="res.qrc">
          <normaloff>:/images/images/QP.png</normaloff>:/images/images/QP.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>10</y>
         <width>111</width>
         <height>31</height>
        </rect>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">image: url(:/images/images/Titel2.png);
font: 12pt &quot;Microsoft YaHei UI&quot;;
color: rgb(255, 255, 255);
QLabel {
    qproperty-alignment: AlignLeft;
}</string>
       </property>
       <property name="text">
        <string>抖音</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QPushButton" name="BtnHome">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>92</width>
         <height>28</height>
        </rect>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;Microsoft YaHei UI&quot;;
color: rgb(255, 255, 255);
</string>
       </property>
       <property name="text">
        <string>首页</string>
       </property>
       <property name="icon">
        <iconset resource="res.qrc">
         <normaloff>:/images/images/Home.png</normaloff>:/images/images/Home.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>25</width>
         <height>25</height>
        </size>
       </property>
      </widget>
      <widget class="QPushButton" name="BtnQQ">
       <property name="geometry">
        <rect>
         <x>660</x>
         <y>10</y>
         <width>92</width>
         <height>28</height>
        </rect>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);
font: 8pt &quot;Microsoft YaHei UI&quot;;</string>
       </property>
       <property name="text">
        <string>私信</string>
       </property>
       <property name="icon">
        <iconset resource="res.qrc">
         <normaloff>:/images/images/Communication.png</normaloff>:/images/images/Communication.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>25</width>
         <height>25</height>
        </size>
       </property>
      </widget>
      <widget class="QPushButton" name="btnAdd">
       <property name="geometry">
        <rect>
         <x>770</x>
         <y>10</y>
         <width>31</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="res.qrc">
         <normaloff>:/images/images/TG.png</normaloff>:/images/images/TG.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>25</width>
         <height>25</height>
        </size>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="BtnZY">
       <property name="geometry">
        <rect>
         <x>810</x>
         <y>10</y>
         <width>41</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">image: url(:/images/images/YQ.jpg);
</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </widget>
      <widget class="QLabel" name="labname">
       <property name="geometry">
        <rect>
         <x>8</x>
         <y>420</y>
         <width>81</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>账号：</string>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="Line" name="line">
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>TMyVideoWidget</class>
   <extends>QWidget</extends>
   <header location="global">tmyvideowidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
